docs/markdown/Dependencies.md

```diff
@@ -64,6 +64,30 @@ pkg-config files. Meson has autodetection support for some of these,
 and they are described [later in this
 page](#dependencies-with-custom-lookup-functionality).
 
+# Arbitrary variables from dependencies that can be found multiple ways
+
+*Note* new in 0.51.0
+
+When you need to get an arbitrary variables from a dependency that can be
+found multiple ways, and you don't want to constrain the type you can use
```

Comment:
Does this need a comma after 'constrain the type' to mark a new clause?  Should this sentence better be in passive voice?